apps/mobile/src/libs/components/wheel/libs/helpers/get-random-value/get-random-value.helper.ts

```diff
@@ -0,0 +1,14 @@
+const DEFAULT_MIN_VALUE = 0;
+const CORRECTION_VALUE = 1;
+
+const getRandomValue = ({
```

Comment:
```suggestion
const getSectorRandomValue = ({
```
Let's be more specific with the name